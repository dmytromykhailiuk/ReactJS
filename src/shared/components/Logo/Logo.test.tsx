import { Logo } from "./";
import { shallow } from "enzyme";
import React from "react";
import toJson from "enzyme-to-json";

describe("Logo", () => {
  it("should match snapshot", () => {
    const wrapper = shallow(<Logo />);
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
