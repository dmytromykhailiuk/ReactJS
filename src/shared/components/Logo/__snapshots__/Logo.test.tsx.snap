// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo should match snapshot 1`] = `
<div
  className="logo"
>
  netflix
  <span
    className="logo__roulette"
  >
    roulette
  </span>
</div>
`;
